### PR TITLE
Fix hands harm

### PR DESCRIPTION
**Pull request by ocelot134269**

### CODE REVIEW

File: magic/divine/healers.cpp [hunk 1]
```diff
@@ -400,11 +411,11 @@ int cmdPray(Player* player, cmd* cmnd) {
         }
     } else {
         if(player->isEffected("prayer")) {
-            player->print("Your spiritual blessing prevents you from praying.\n");
+            player->print("Your current spiritual blessing prevents you from praying.\n");
```
Comment (mitchej123) [inline magic/divine/healers.cpp:414]: psh, kill these prints!

**Review by mitchej123 — APPROVED** [review]

File: magic/divine/healers.cpp (hunk 1 above)
Comment (ocelot134269) [reply magic/divine/healers.cpp]: re-wrote cmdPray entirely, removed all print()

File: combat/warriors.cpp [hunk 2]
```diff
@@ -1049,47 +1046,55 @@ int cmdHarmTouch(Player* player, cmd* cmnd) {
 
 
 
-    creature->print("%M hits you with fists shrouded in darkness!\n", player);
-    broadcast(player->getSock(),  creature->getSock(), creature->getRoomParent(), "%M hits %N with fists shrouded in darkness!", player, creature);
+    *creature << player << " touches you with " << player->hisHer() << " malevolent hand!\n";
```
Comment (mitchej123) [inline combat/warriors.cpp:1049]: this probably needs setf(CAP); (or whatever it is) try it with a possessed mob on live vs after this change.

File: effects/effectsLoader.cpp [hunk 3]
```diff
@@ -990,9 +990,9 @@ bool Config::loadEffects() {
         .applyScript("actor.addStatModEffect(effect)")
         .unApplyScript("actor.remStatModEffect(effect)")
         .selfAddStr("^yYou feel extremely pious.^x")
-        .roomAddStr("^y*ACTOR* bows *A-HISHER* head in prayer.^x")
+        .roomAddStr("^y*ACTOR* is more pious.^x")
```
Comment (mitchej123) [inline effects/effectsLoader.cpp:993]: cough.  Looks more pious?

**Pull request merged**